pkg/cli/connect_helm.go

```diff
@@ -88,8 +88,16 @@ func ConnectHelm(ctx context.Context, options *ConnectOptions, globalFlags *flag
 }
 
 func (cmd *connectHelm) connect(ctx context.Context, vCluster *find.VCluster, command []string) error {
+	currentContext, _, err := find.CurrentContext()
+	if err != nil {
+		return err
+	}
+	if currentContext == find.VClusterContextName(vCluster.Name, vCluster.Namespace, vCluster.Context) {
+		return fmt.Errorf("already connected to vcluster %q", vCluster.Name)
```

Comment:
I'm not sure if printing an error here is the right choice as that would prevent this correct:
```
vcluster connect vcluster => works
vcluster connect vcluster => error
```
I think it's fine to not do certain things on the second connect but I think the command should suceed as otherwise this leads to a worse UX for the user